porch/controllers/workloadidentitybinding/pkg/controllers/workloadidentitybinding/controller.go

```diff
@@ -45,46 +46,56 @@ type WorkloadIdentityBindingReconciler struct {
 //+kubebuilder:rbac:groups=config.porch.kpt.dev,resources=workloadidentitybindings/finalizers,verbs=update
 //+kubebuilder:rbac:groups=,resources=namespaces,verbs=get;list;watch
 //+kubebuilder:rbac:groups=iam.cnrm.cloud.google.com,resources=iampolicymembers,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups=,resources=serviceaccounts,verbs=get;list;watch;create;update;patch;delete
 
 // Reconcile implements the main kubernetes reconciliation loop.
 func (r *WorkloadIdentityBindingReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
 	var subject api.WorkloadIdentityBinding
 	if err := r.Get(ctx, req.NamespacedName, &subject); err != nil {
 		return ctrl.Result{}, client.IgnoreNotFound(err)
 	}
-	// myFinalizerName := "config.porch.kpt.dev/finalizer"
-	// if subject.ObjectMeta.DeletionTimestamp.IsZero() {
-	// 	// The object is not being deleted, so if it does not have our finalizer,
-	// 	// then lets add the finalizer and update the object. This is equivalent
-	// 	// registering our finalizer.
-	// 	if !controllerutil.ContainsFinalizer(&subject, myFinalizerName) {
-	// 		controllerutil.AddFinalizer(&subject, myFinalizerName)
-	// 		if err := r.Update(ctx, &subject); err != nil {
-	// 			return ctrl.Result{}, err
-	// 		}
-	// 	}
-	// } else {
-	// 	// The object is being deleted
-	// 	if controllerutil.ContainsFinalizer(&subject, myFinalizerName) {
-	// 		// // our finalizer is present, so lets handle any external dependency
-	// 		// if err := r.deleteExternalResources(ctx, &subject); err != nil {
-	// 		// 	// if fail to delete the external dependency here, return with error
-	// 		// 	// so that it can be retried
-	// 		// 	return ctrl.Result{}, fmt.Errorf("have problem to delete external resource: %w", err)
-	// 		// }
-	// 		// remove our finalizer from the list and update it.
-	// 		controllerutil.RemoveFinalizer(&subject, myFinalizerName)
-	// 		if err := r.Update(ctx, &subject); err != nil {
-	// 			return ctrl.Result{}, fmt.Errorf("failed to update %s after delete finalizer: %w", req.Name, err)
-	// 		}
-	// 	}
-	// 	// Stop reconciliation as the item is being deleted
-	// 	return ctrl.Result{}, nil
-	// }
+	myFinalizerName := "config.porch.kpt.dev/finalizer"
```

Comment:
1) should this be a package level const?
2) do we use `config.porch.kpt.dev` elsewhere? Why not just `porch.kpt.dev`?

Reply:
Yeah, moved this to a package level const.
We are using `config.porch.kpt.dev` group for CRDs since the `porch.kpt.dev` group is already in use by the aggregated apiserver.